.github/workflows/test.yml

```diff
@@ -34,6 +39,17 @@ jobs:
       - name: cargo test --doc
         run: cargo test --locked --all-features --doc
   minimal:
+    # This action chooses the oldest version of the dependencies to ensure that this crate is
+    # compatible with the minimal version that this crate and its dependencies require.
+    # This will pickup issues where this create relies on functionality that was introduced
+    # later than the actual version specified (e.g. when we choose just a major version, but a
+    # method was added after this version).
+    #
+    # This particular check can be difficult to get to succeed as often transitive dependencies
+    # may be incorrectly specified (e.g. a dependency specifies 1.0 but really requires 1.1.5).
+    # There is an alternative flag available -Zminimal-direct that uses the minimal versions for
+    # direct dependencies of this crate, while selecting the maximal versions for the transitive
+    # dependencies
```

Comment:
> Alternatively, you can add a line in your `Cargo.toml` to artificially increase the minimal dependency, which you do with:
> ```toml
> # for minimal-versions
> [target.'cfg(any())'.dependencies]
> openssl = { version = "0.10.55", optional = true } # needed to allow foo to build with -Zminimal-versions
> ```

The `optional = true` is necessary in case that dependency isn't otherwise transitively required by your library, and the `target` bit is so that this dependency edge never actually affects Cargo build order.

See also https://github.com/jonhoo/fantoccini/blob/fde336472b712bc7ebf5b4e772023a7ba71b2262/Cargo.toml#L47-L49.